### PR TITLE
[Terra] Update terra block explorer

### PR DESCRIPTION
**Pull request by hewigovens**
Latest will be columbus-5 https://twitter.com/terra_money/status/1432927061849427968

### CODE REVIEW

**Review by vikmeup — APPROVED** [review]
So there is no way to open without specifying chain ID?

**Review by vikmeup — APPROVED** [review]

**Review by catenocrypt — APPROVED** [review]

**Comment by hewigovens** [comment]
don't merge this yet, Terra team might add url redirect support later

**Comment by hewigovens** [comment]
this still works after columbus-5, let's merge it anyway

**Pull request merged**
